lib/converters/v2.1.0/converter-v21-to-v1.js

```diff
@@ -0,0 +1,130 @@
+var _ = require('lodash').noConflict(),
+    util = require('../../util'),
+    inherits = require('inherits'),
+
+    v2Common = require('../common/v2'),
+    BaseBuilders = require('../v2.0.0/converter-v2-to-v1').Builders,
+
+    Builders;
+
+inherits(Builders = function ConversionBuilder () {
+    Builders.super_.apply(this, arguments);
+}, BaseBuilders);
+
+_.extend(Builders.prototype, {
+    /**
+     * Converts a V2.1 request to a V1 request.
+     *
+     * @param {Item|Object} item - The wrapped request instance to be converted.
+     * @param {String} collectionId - The collection id to be added to the converted v1.0.0 compliant request.
+     * @param {Boolean} skipResponses - Will result in saved responses being ignored for conversion if set to true.
+     * @returns {Object} An equivalent v1.0.0 representation for the current v2.1.0 request.
+     */
+    request: function (item, collectionId, skipResponses) {
+        var auth = _.get(item, 'request.auth');
+
+        if (auth && auth.type && _.isArray(auth[auth.type])) {
+            auth[auth.type] = _.transform(auth[auth.type], function (a, element) {
```

Comment:
add todo to later handle non `type` and non underscore parameters

Reply:
Addressed